x-pack/filebeat/docs/inputs/input-gcs.asciidoc

```diff
@@ -276,6 +280,8 @@ filebeat.inputs:
     - regex: '/Security-Logs/'
 ----
 
+NOTE: The `file_selectors` op is performed within the agent locally which scales vertically, hence using this option will cause the agent to download all the files and then filter them. This can cause a bottleneck in processing if the number of files is very high. It is recommended to use this attribute only when the number of files is limited or ample resources are available.
```

Comment:
```suggestion
The `file_selectors` op is performed within the agent locally which scales vertically, hence using this option will cause the agent to download all the files and then filter them. This can cause a bottleneck in processing if the number of files is very high. It is recommended to use this attribute only when the number of files is limited or ample resources are available.
```
I'm not convinced that horizontal v vertical scaling is meaningful in this context.